src/embed-code/embed-code.const.ts

```diff
@@ -0,0 +1,62 @@
+import bookwidgetLogo from '@assets/images/bookwidget_logo.png';
+import smartschoolLogo from '@assets/images/smartschool_logo.png';
+
+import type { FilterableColumn } from '../admin/shared/components/FilterTable/FilterTable';
+import { isMobileWidth } from '../shared/helpers';
+import { tText } from '../shared/helpers/translate-text';
+
+import type { EmbedCodeOverviewTableColumns } from './embed-code.types';
+
+export const BookWidgetsLogo = bookwidgetLogo;
+export const SmartSchoolLogo = smartschoolLogo;
```

Comment:
usually capital variables are used for types or components, not for strings
maybe this works?

if not, it might be better to import and export like:
```
import bookwidgetLogoInternal from '@assets/images/bookwidget_logo.png';
import smartschoolLogoInternal from '@assets/images/smartschool_logo.png';
export const bookWidgetsLogo = bookwidgetLogoInternal;
export const smartSchoolLogo = smartschoolLogoInternal;
```

```suggestion
export bookwidgetLogo from '@assets/images/bookwidget_logo.png';
export smartschoolLogo from '@assets/images/smartschool_logo.png';

import type { FilterableColumn } from '../admin/shared/components/FilterTable/FilterTable';
import { isMobileWidth } from '../shared/helpers';
import { tText } from '../shared/helpers/translate-text';

import type { EmbedCodeOverviewTableColumns } from './embed-code.types';
```